### DISCUSSION TITLE
Botton for "Show Desktop" as in Windows 10/11

### DISCUSSION THREAD

**Discussion by Anthropos888** [lawand-dot-io/taskbar#46] [Feature requests]
Please add a button on the rigth to show the desktop, as in Windows 10/11.
Currently there is no way to show the desktop when Taskbar is running.

On MacOS 14 you can just click on the Desktop left/right to the dock.
But in Taskbar you cannot do that since the Taskbar is maximized in length in the desktop is not visible.

So you need a button on the right or left to show the desktop.

**Comment by lawand-dot-io** [comment]
Hello. Thanks for the feature request. There's a similar request so I'll close this one for simplicity. Feel free to upvote the other request to increase the chance I implement it sooner :) https://github.com/lawand-dot-io/taskbar/discussions/25
